src/miner.cpp

```diff
@@ -312,35 +308,32 @@ CBlockTemplate* CreateNewBlock(const CChainParams& chainparams, const MinerAddre
 
                 int nConf = nHeight - coins->nHeight;
 
-                dPriority += (double)nValueIn * nConf;
             }
             nTotalIn += tx.GetShieldedValueIn();
 
             if (fMissingInputs) continue;
 
-            // Priority is sum(valuein * age) / modified_txsize
             unsigned int nTxSize = ::GetSerializeSize(tx, SER_NETWORK, PROTOCOL_VERSION);
-            dPriority = tx.ComputePriority(dPriority, nTxSize);
 
+            double nDummyPriority = 0;
             uint256 hash = tx.GetHash();
-            mempool.ApplyDeltas(hash, dPriority, nTotalIn);
+            mempool.ApplyDeltas(hash, nDummyPriority, nTotalIn);
 
             CFeeRate feeRate(nTotalIn-tx.GetValueOut(), nTxSize);
 
             if (porphan)
             {
-                porphan->dPriority = dPriority;
                 porphan->feeRate = feeRate;
             }
             else
-                vecPriority.push_back(TxPriority(dPriority, feeRate, &(mi->GetTx())));
+                vecPriority.push_back(TxPriority(nDummyPriority, feeRate, &(mi->GetTx())));
         }
 
         // Collect transactions into block
         uint64_t nBlockSize = 1000;
         uint64_t nBlockTx = 0;
         int nBlockSigOps = 100;
-        bool fSortedByFee = (nBlockPrioritySize <= 0);
+        bool fSortedByFee = 1;
```

Comment:
Two things: using numeric values for booleans is kinda gross, and if we're hardcoding this value we should instead see if it can be removed.